test/playlists.test.js

```diff
@@ -0,0 +1,107 @@
+import QUnit from 'qunit';
+import { makeMochTech } from './util/util.js';
+import { FlashlsHandler } from '../src/index.js';
+
+QUnit.module('Flashls playlists');
+
+QUnit.test('triggers an event when the active media changes', function(assert) {
+  let mediaChange = 0;
+  const levels = [
```

Comment:
Since this test is just testing the event forwarding, you can remove anything to do with levels/currentLevel. Technically the swf would have a new currentLevel when it does a levelswitch, but since we are faking it, we don't need to fake that part as well